HW-11/Homework-11.py

```diff
@@ -0,0 +1,60 @@
+# decorator for any function:
+def my_new_decorator(func):
+    def the_wrapper_around_the_original_function(*args, **kwargs):
+        from datetime import datetime
+        print(func.__name__, datetime.now().strftime('%H:%M'))
+        func(*args, **kwargs)
+
+    return the_wrapper_around_the_original_function
+
+
+# Custom Exception class:
+class MyCustomException(Exception):
+    pass
+
+
+# raise MyCustomException("Custom exception is occured")
+
+# Context manager
+class MyWithManag(object):
+    def __init__(self, func):
```

Comment:
You are taking this `func` argument but it is unused. You can remove it since you don't need it. And you can skip `as r` part when using this context manager since you don't return any value here